### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by matthiasbeyer**

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@neithernut You could enable build-abort on travis, so we only build the latest commit of each PR. Would speedup the build process for a PR.

File: README.md [hunk 1]
```diff
@@ -1,42 +1,42 @@
 git-dit - the distributed issue tracker for git
 
-This repository contains the "git-dit" project, a distributed issue tracking
-system using git. For a system overview and conceptual information, refer to the
-[documentation](doc/README.md). For a more practical documentation, refer to the
-[man page](git-dit.1.md).
+[![Build Status](https://travis-ci.org/neithernut/git-dit.svg?branch=master)](https://travis-ci.org/neithernut/git-dit)
+[![Issue Stats](https://img.shields.io/badge/issues%20closed%20in-7%20days-yellowgreen.svg?style=flat-square)](http://www.issuestats.com/github/neithernut/git-dit)
+[![](https://tokei.rs/b1/github/neithernut/git-dit)](https://github.com/neithernut/git-dit)
+[![license](https://img.shields.io/github/license/neithernut/git-dit.svg?maxAge=2592000?style=flat-square)]()
```
Comment (neithernut) [inline README.md]: I don't know whether I told you before, but I'm in strong opposition to such badges. For example:
 * They ate specific to github.
 * The README may not only be read on github but also offline. In such a situation, the badges are rather annoying.
 * The state the badges are displaying is the current state, not the state of the commit which may be viewed.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+* Distributed issue tracking, without checking files into the repository
+* Convenient commandline interface (plumbing + porcelain)
+* Implemented as `git` subcommand: `git dit`
+* Each command has a `--help`
+* No structured data 
+  * An issue/comment is a commit
+  * "Tags" are supported (see `man git-interpret-trailers`)
+* Convenience `git dit push` and `git dit pull`
+* No additional software needed on the server-side. Pushing/Pulling from
+  github, gitlab, gitolite should work out of the box
```
Comment (neithernut) [inline README.md]: This implicitly advertises a conversion to github- and gitlab-issues.

**Comment by neithernut** [comment]
Not sure whether anything is missing/should be added to the README at this point. For example, we could add some basic installation info (based on a working cargo setup, not considering dependencies, ...)

**Comment by neithernut** [comment]
@matthiasbeyer do you want to add or squash something? E.g. 37d1226cd672b96d43b2c09258923bc90059f39b and bf4320fd1885c3750a24a7bdcbe158da83298e99?

**Pull request merged**